README.md

```diff
@@ -46,12 +46,14 @@ that your code fulfills these standards before any Pull Request.
 
 ``` bash
 composer.phar update
-bin/php-cs-fixer fix
-bin/php-formatter formatter:use:sort src/
-bin/php-formatter formatter:header:fix src/
+composer.phar check-style
```

Comment:
If we consider composer as a system-wide script, here just `composer` should be used (preferred), if not I would suggest a `php composer.phar`.